alerts/hadoop/README.md

```diff
@@ -0,0 +1,24 @@
+# Alerts for Hadoop in the Ops Agent
+
+### Notification Channels
+For all alerts, a notification channel needs to be set up or the alert will fire silently.
+
+### User Labels
+User labels can be used for these policies by modifying the userLabels fields of the policies. i.e.
+
+```json
+{ 
+  "userLabels": {
+    "datacenter": "central"
+  }
+}
+```
+
+## Low available capacity alert
+A low available capacity means that the total available disk space capacity across all HDFS clusters is low and should be increased.
```

Comment:
These descriptions are great, thank you. Could you also include what threshold is being used in the alert policy sample? i.e. how is "low" being defined?

Reply:
added in threshold details and organized the section